src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/ReviewCodeForSqlInjectionVulnerabilitiesTests.cs

```diff
@@ -622,6 +624,40 @@ protected void Page_Load(object sender, EventArgs e)
                 GetCSharpResultAt(15, 37, 15, 52, "SqlCommand.SqlCommand(string cmdText)", "void WebForm.Page_Load(object sender, EventArgs e)", "string HttpRequest.this[string key]", "void WebForm.Page_Load(object sender, EventArgs e)"));
         }
 
+        [Fact]
+        public async Task HttpRequest_Form_Method_GenericsSink_Diagnostic()
+        {
+            var csharpTest = new VerifyCS.Test
+            {
+                ReferenceAssemblies = ReferenceAssemblies.NetFramework.Net472.Default
+                    .AddAssemblies(ImmutableArray.Create("System.Web", "System.Web.Extensions"))
```

Comment:
```suggestion
                ReferenceAssemblies = AdditionalMetadataReferences.DefaultWithSystemWeb
```

Reply:
I have copy/pasted it from the Master branch. You'll get a conflict on merge otherwise.